src/main/java/seedu/address/model/attendance/Hours.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.model.attendance;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the number of hours a Member has gone for an Event.
+ * Guarantees: immutable; is valid as declared in {@link #isValidHours(String)}
+ */
+public class Hours {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Hours should be an integer and it should not be blank";
+
+    public final Integer value;
+
+    /**
+     * Constructs a {@code Hours}.
+     *
+     * @param hours A valid number of hours in String.
+     */
+    public Hours(String hours) {
+        requireNonNull(hours);
+        checkArgument(isValidHours(hours), MESSAGE_CONSTRAINTS);
+        value = Integer.parseInt(hours);
+    }
+
+    /**
+     * Returns true if a given number of hours is a valid hours.
+     */
+    public static boolean isValidHours(String test) {
+        try {
+            Integer.valueOf(test);
```

Comment:
Should we test for negative values and include it in the constrains? I believe users can enter negative numbers right now.

Reply:
I agree. We should ensure that we do not allow negative values.

Reply:
Good suggestion, I will add that